app/scripts/directives/overview/virtualMachineRow.js

```diff
@@ -97,6 +99,16 @@
     };
   }
 
+  angular.module('openshiftConsole').filter('podUptime', function () {
+    return function (pod) {
+      var startTime = _.get(pod, 'status.startTime');
```

Comment:
Do we want to get the actual container start time?

I think if the container is crash looping, it will still count as "up" since the pod is running.

Reply:
Fixed, thanks.